app/controllers/gallery/albums_controller.rb

```diff
@@ -0,0 +1,12 @@
+# encoding:UTF-8
+class Gallery::AlbumsController < ApplicationController
+  load_permissions_and_authorize_resource
+
+  def show
+    if params[:start].present?
+      @start = params[:start]
+    elsif params[:image].present?
+      @start = @album.images.find_index(Image.find(params[:image]))
+    end
```

Comment:
Hmm, can you elaborate? If the image is not in the collection?

This code is rather ugly, but since the gallery is javascript we need 
the ability to link to a certain starting index. Any suggestions are very welcome :)